src/main/java/proj/kedabra/billsnap/business/service/impl/CalculatePaymentServiceImpl.java

```diff
@@ -59,6 +59,11 @@ public BigDecimal calculateTip(final BigDecimal billTipAmount, final BigDecimal
         return tipAmount.add(tipPercentAmount).setScale(DOLLAR_SCALE, RoundingMode.HALF_UP);
     }
 
+    @Override
+    public BigDecimal calculateAmountRemaining(final BigDecimal total, final BigDecimal amountPaid) {
+        return total.subtract(amountPaid);
```

Comment:
missing tests

Reply:
how did you spot there was missing tests? It said 100% coverage

Reply:
I looked over your code and didn't see a calservicetest class